config/queue.php

```diff
@@ -63,10 +63,9 @@
             'driver' => 'redis',
             'connection' => 'default',
             'queue' => env('REDIS_QUEUE', 'default'),
-            'retry_after' => 90,
+            'retry_after' => 60 * 60 * 6,
```

Comment:
Instead of raising these values globally and still having each item processed on the `default` connection, we could also introduce a dedicated connection for MediaWiki jobs. This would mean we'd need to add an extra deployment for that queue in the `api` Helm chart.